opentelemetry-sdk/src/opentelemetry/sdk/correlationcontext/__init__.py

```diff
@@ -15,48 +15,46 @@
 import typing
 from contextlib import contextmanager
 
-from opentelemetry import distributedcontext as dctx_api
+from opentelemetry import correlationcontext as dctx_api
 from opentelemetry.context import Context
 
 
-class DistributedContextManager(dctx_api.DistributedContextManager):
-    """See `opentelemetry.distributedcontext.DistributedContextManager`
+class CorrelationContextManager(dctx_api.CorrelationContextManager):
+    """See `opentelemetry.correlationcontext.CorrelationContextManager`
 
     Args:
         name: The name of the context manager
     """
 
     def __init__(self, name: str = "") -> None:
         if name:
-            slot_name = "DistributedContext.{}".format(name)
+            slot_name = "CorrelationContext.{}".format(name)
         else:
-            slot_name = "DistributedContext"
+            slot_name = "CorrelationContext"
 
         self._current_context = Context.register_slot(slot_name)
 
-    def get_current_context(
-        self,
-    ) -> typing.Optional[dctx_api.DistributedContext]:
-        """Gets the current DistributedContext.
+    def current_context(self,) -> typing.Optional[dctx_api.CorrelationContext]:
```

Comment:
The verb `get` is being removed from the function name, which is fine. Nevertheless, I would recommend to make this function a property then, in order to have an API that resembles an attribute being accessed (`current_context`) instead of an action being performed (`current_context()`).